task.sh

```diff
@@ -6,38 +6,59 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir -p task/dir1 task/dir2 task/dir3 task/dir3/dir4
```

Comment:
При использовании флага `-p` можно не указывать промежуточные папки
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```